ui/pages/citizen.tsx

```diff
@@ -17,6 +18,19 @@ export default function Citizen() {
     setConfettiNumber([...confettiNumber, confettiNumber.length])
   }
 
+  const { signMessage: signMessageAndDownloadPass } = useSignMessage({
+    message: 'I am the holder of this Nation3 passport',
+    onSuccess(data) {
+      console.log('signMessageAndDownloadPass data:', data)
+      const downloadPassURI : string = `https://mobile-passport.vercel.app/api/downloadPass?address=${account.address}&signature=${data}&platform=Apple`
```

Comment:
@xPi2 Is `mobile-passport.vercel.app` the correct domain?  Could you check in Vercel if there is another domain configured?